sdk/v2/core/events.go

```diff
@@ -249,6 +249,9 @@ type EventsClient interface {
 	// DeleteMany deletes multiple Events specified by the EventListOptions
 	// parameter.
 	DeleteMany(context.Context, EventsSelector) (DeleteManyEventsResult, error)
+	// Retry copies an Event, including Worker configuration and Jobs, and
+	// creates a new Event from this information.
```

Comment:
May want to update the comment to reflect that "where possible" successful job results are reused and are not re-run.